linkerd/app/outbound/src/http/logical/policy/route/metrics/labels.rs

```diff
@@ -1,21 +1,29 @@
 //! Prometheus label types.
-use linkerd_app_core::{errors, metrics::prom::EncodeLabelSetMut, proxy::http, Error as BoxError};
+use linkerd_app_core::{
+    dns, errors, metrics::prom::EncodeLabelSetMut, proxy::http, Error as BoxError,
+};
 use prometheus_client::encoding::*;
 
 use crate::{BackendRef, ParentRef, RouteRef};
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq)]
-pub struct Route(pub ParentRef, pub RouteRef);
+pub struct HttpRoute {
+    parent: ParentRef,
+    route: RouteRef,
+    name: Option<dns::Name>,
```

Comment:
'name' can be a bit ambiguous in the context of our code... especially with regard to identity vs hosts vs ...

```suggestion
    hostname: Option<dns::Name>,
```